### PR TITLE
feat: booking confirmation event

### PR DESCRIPTION
**Pull request by fandihungryhub**
https://3.basecamp.com/5190892/buckets/31862334/card_tables/cards/6038383000

### CODE REVIEW

File: components/booking/confirmation/BookingHeader.vue [hunk 1]
```diff
@@ -4,9 +4,12 @@
       <button type="button" @click="onClickBackButton">
         <IcChevronLeft class="text-base" />
       </button>
-      <NuxtLink v-if="props.shareLink" :to="props.shareLink"
+      <!--NuxtLink v-if="props.shareLink" :to="props.shareLink"
```
Comment (afif-hh) [inline components/booking/confirmation/BookingHeader.vue:7]: dihapus

File: pages/hybrid/confirmation.vue [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  listenFromPlatform({
+    eventName: EVENT_DO_SETUP_DATA,
+    callback: async (data: SetupDataPayload) => {
+      try {
+        const parsedData = !isWebClient ? convertToJsonObject(data) : data;
+        setupDataSchema.parse(parsedData);
+        dataReceived(parsedData);
+        isLoading.value = false;
+      } catch (err) {
+        setupDataValidation.error = true;
```
Comment (afif-hh) [inline pages/hybrid/confirmation.vue:78]: dikasih useHandleError mas

**Review by afif-hh — CHANGES_REQUESTED** [review]

File: components/booking/confirmation/BookingHeader.vue (hunk 1 above)
Comment (fandihungryhub) [reply components/booking/confirmation/BookingHeader.vue]: updated

File: pages/hybrid/confirmation.vue (hunk 2 above)
Comment (fandihungryhub) [reply pages/hybrid/confirmation.vue:79]: updated

**Review by afif-hh — APPROVED** [review]

**Pull request merged**